app/views/application/_subnavigation.html.erb

```diff
@@ -7,9 +7,13 @@
 
   <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-0 govuk-!-padding-right-0">
     <nav class="govuk-body govuk-!-margin-bottom-0 ">
+    <% if current_user.admin? %>
+      <%= link_to "Organisations", organisations_path, class: active_tab(organisations_path) %>
+    <% else %>
```

Comment:
This `else` means that if you are an admin you don't see any of the other features. Is that intended?

Reply:
Yeah. Jakub suggested that as admins we may not need the other features.